hr_planning/model/hr_planning_day.py

```diff
@@ -0,0 +1,31 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: David Coninckx <david@coninckx.com>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from openerp import fields, models, _
+
+
+class hr_planning_day(models.Model):
+    _name = "hr.planning.day"
+    '''
+        Simple model for a working day for an employee.
+    '''
+    employee_id = fields.Many2one(
+        'hr.employee', 'Employee', readonly=True)
+    start_date = fields.Datetime('Start date', readonly=True)

```

Comment:
In that case, the field name is not needed ('Start date'), because Odoo will automatically generate it correctly based on the field_name (start_date).